projects/components/src/search-box/search-box.component.ts

```diff
@@ -111,11 +111,14 @@ export class SearchBoxComponent implements OnInit, OnChanges {
   public searchMode: SearchBoxEmitMode = SearchBoxEmitMode.Incremental;
 
   @Input()
-  public enableSearchHistory: boolean = false; // Experimental
+  public enableSearchHistory: boolean = true; // Experimental
 
   @Input()
   public collapsable: boolean = false;
 
+  @Input()
+  public searchHistoryDebounceTime: number | undefined;
```

Comment:
We should use the existing debounce time.

Reply:
So, we should remove this as an Input.

Reply:
Removed this input